packages/yoastseo/src/languageProcessing/researches/getKeywordDensity.js

```diff
@@ -23,7 +21,22 @@ export default function( paper, researcher ) {
 		return 0;
 	}
 
-	const keywordCount = researcher.getResearch( "keywordCount" );
+	const keyphraseCount = researcher.getResearch( "keyphraseCount" );
+
+	return ( keyphraseCount.count / wordCount ) * 100;
+}
 
-	return ( keywordCount.count / wordCount ) * 100;
+/**
+ * Calculates the keyphrase density.
+ *
+ * @deprecated Since version 20.8. Use getKeyphraseDensity instead.
+ *
+ * @param {Object} paper        The paper containing keyphrase and text.
+ * @param {Object} researcher   The researcher.
+ *
+ * @returns {Object} The keyphrase density.
+ */
+export function getKeywordDensity( paper, researcher ) {
+	console.warn( "This function is deprecated, use getKeyphraseDensity instead." );
+	return getKeywordDensity( paper, researcher );
```

Comment:
FYI I expect that because of the [changes in another PR](https://github.com/Yoast/wordpress-seo/pull/20185/files#diff-2f9610db61960b7f0b9cba5283ba38c1b7200a164df2312203d2d2b32b1ba27b), there will be at some point merge conflicts in this file after pulling from the feature branch. When that happens, you can reject the changes from the linked PR. Removing HTML elements from the paper will be no longer needed when this file is adapted to use the HTML parser.